code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -174,3 +174,101 @@
 		T.Entered(src)
 
 //Ayy lmao
+
+// Robotic Tongue emotes. Beep!

```

Comment:
I just wanted to leave a comment that this particular block of code actually breaks silicon emotes, and required me to redo them a bit. 
https://github.com/OracleStation/OracleStation/pull/694 & https://github.com/OracleStation/OracleStation/pull/750